coveo-eks/_locals.tf

```diff
@@ -0,0 +1,33 @@
+locals {
+  kiam_worker_groups = [{
```

Comment:
Wouldnt we want all those parameter to be configurable by an user of the module ?

Reply:
No we don't.